### DISCUSSION TITLE
How torch.ops.pyg.segment_matmul works?

### DISCUSSION THREAD

**Discussion by 1049451037** [pyg-team/pytorch_geometric#6322] [Q&A]
Hi, there is a line of code in pyg_lib:

```python
return torch.ops.pyg.segment_matmul(inputs, ptr, other)
```

How does this work? Where can I find the source code of this function?

**Comment by LukeLIN-web** [comment]
See https://github.com/pyg-team/pyg-lib/blob/master/pyg_lib/csrc/ops/matmul.h
